tests/test_fiod.sh

```diff
@@ -4,35 +4,33 @@ set -xeEo pipefail
 source tests/common.sh
 
 function finish {
-  if [ $? -eq 1 ] && [ $ERRORED != "true" ]
-  then
-    error
-  fi
-
-  echo "Cleaning up Fio"
-  kubectl delete -f tests/test_crs/valid_fiod.yaml
-  delete_operator
+  kubectl delete all --all -n my-ripsaw
```

Comment:
why should this be specific to test_fiod?  and why "delete all --all" instead of just "delete namespace my-ripsaw"?   Just curious, I'm not going to argue about it.

Reply:
If we delete the namespace here. In case of error, the CI would fail in the next retry, since the namespace is deleted/created outside out form the retry loop.
Look here:
https://github.com/cloud-bulldozer/ripsaw/blob/master/run_test.sh#L19-L56

Reply:
true, I'm ok with this as long as it deletes things in the right order, and finalizers should enforce that anyway.   Never mind ;-)

Reply:
I'm making some changes to the namespace create/delete per https://github.com/cloud-bulldozer/ripsaw/pull/271 which adds a namespace delete to wait clean and creation one of the operator functions. I'd also say given that we should just use wait clean here and not just the straight delete all command.